Client/Application/ClientPreferences.swift

```diff
@@ -50,6 +50,9 @@ extension Preferences {
         ///
         /// Currently unused.
         static let showClipboardBar = Option<Bool>(key: "general.show-clipboard-bar", default: false)
+        /// Whether or not user onboarding has completed.
+        /// User skipping onboarding counts as completed too.
+        static let onboardingCompleted = Option<Bool>(key: "general.onboarding-completed", default: false)
```

Comment:
We may eventually have multiple types of onboardings (e.g. rewards), and although we don't "now", I'm thinking we should maybe future proof ourselves a bit more. Maybe: `onboardingBasicCompleted` or `Initial`?

Like eventually when we ship rewards, new users may get both onboardings tied together (looks like one), but existing users will just get the rewards screens (as they would have already seen the "basic" one here).